### ISSUE TITLE
[Accessibility] Close button in banner is not focusable for keyboard users

### ISSUE THREAD

**Issue by DBowen33** [microsoft/vscode#278799]
PROBLEM:
The close button in banners is not focusable for keyboard users.

OBSERVED RESULTS:
When a banner is present, keyboard users are not able to focus on the close button of the banner.

EXPECTED RESULTS:
Close button should be focusable so that keyboard only users are able to close the banner.

USER IMPACT:
Keyboard only users do not have a way of interacting with the close button of the banner.

STEPS TO REPRODUCE:

1. Open VSCode with banner present
2. Use the keyboard to tab to banner
3. Notice that the close button is not reachable by keyboard

ENVIRONMENT DETAILS:
Machine: Chromebook
Browser: Version 136.0.7103.150 (Official Build) (64-bit)
Accessibility Tool: ChromeVox.
Bug Filling Time: 20 mins.

Machine: MacBook Pro
Browser: Version 137.0.7151.104 (Official Build) (arm64)
Accessibility Tool: VoiceOver.
Bug Filling Time: 20 mins."

**Comment by meganrogge** [comment]
We don't need tab index for this button as it's included in a toolbar. Arrow keys are used to navigate the toolbar all over the workbench.
